tests/api.c

```diff
@@ -5940,14 +6002,15 @@ static int test_wc_AesCbcEncryptDecrypt (void)
     }
         printf(resultFmt, cbcDWK == 0 ? passed : failed);
 #endif
-    return 0;
+    return ret;
```

Comment:
Is returning "ret" correct here?  Looks like this section of code also uses "cbcE", "cbcD", and "cbcDWK" to hold error state.